# timestamp
Header header

# contact and over-forced judgement
bool is_contact
bool is_overforced
